airbyte-proxy/run.sh

```diff
@@ -16,7 +16,7 @@ else
   TEMPLATE_PATH="/etc/nginx/templates/nginx-auth.conf.template"
 fi
 
-envsubst '${PROXY_PASS_WEB} ${PROXY_PASS_API} ${PROXY_PASS_RESOLVER}' < $TEMPLATE_PATH > /etc/nginx/nginx.conf
+envsubst '${PROXY_PASS_WEB} ${PROXY_PASS_API} ${CONNECTOR_BUILDER_SERVER_API} ${PROXY_PASS_RESOLVER}' < $TEMPLATE_PATH > /etc/nginx/nginx.conf
```

Comment:
search and replace the patterns with the value of the corresponding bash variable
https://www.baeldung.com/linux/envsubst-command#:~:text=The%20envsubst%20command%20searches%20the,envsubst%20recognizes%20only%20exported%20variables.